test/e2e/serial/test_suite_serial_test.go

```diff
@@ -34,8 +59,105 @@ import (
 // and indisturbed on the cluster. No concurrency at all is possible,
 // each test "owns" the cluster - but again, must leave no leftovers.
 
-func TestSerial(t *testing.T) {
+// do not use this fixture outside this *file*
+var __fxt *e2efixture.Fixture
+
+var _ = BeforeSuite(func() {
+	// this must be the very first thing
 	rand.Seed(time.Now().UnixNano())
+
+	var err error
+
+	__fxt, err = e2efixture.Setup("e2e-test-infra")
+	Expect(err).ToNot(HaveOccurred(), "unable to setup infra test fixture")
+
+	nroSchedObj = &nropv1alpha1.NUMAResourcesScheduler{}
+	err = __fxt.Client.Get(context.TODO(), client.ObjectKey{Name: nrosched.NROSchedObjectName}, nroSchedObj)
+	Expect(err).ToNot(HaveOccurred(), "cannot get %q in the cluster", nrosched.NROSchedObjectName)
+
+	nroOperObj = &nropv1alpha1.NUMAResourcesOperator{}
```

Comment:
At what stage during the serial test we are supposed to deploy the ```NUMAResourcesScheduler``` and ```NUMAResourcesOperator```? is the ```install``` suit going to be a prerequisite for the serial suit?

Reply:
This is something I need to document better: the `serial` suite is expected to run against a fully configured TAS-enabled cluster (no matter how) and to validate it. We don't need an explicit dep on the `install` suite, as `install` suite is just one of the many vehicles to deploy TAS on a cluster.

Reply:
OK but it's something we should bear in mind, especially because I guess we'll have a dedicated lane for this type of test.

Reply:
absolutely, added a README to document the use case and the goals of the lane